src/NuGet/Microsoft.CodeAnalysis.CSharp.Features.nuspec

```diff
@@ -4,6 +4,8 @@
     <id>Microsoft.CodeAnalysis.CSharp.Features</id>
     <description>
```

Comment:
I think I'll just move the existing description tags to summary then and add the new info as the body of description.